prow/manifests/overlays/metal3/config.yaml

```diff
@@ -731,50 +731,146 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
-  # name: {job_prefix}-{image_os}-e2e-feature-test-{capm3_target_branch}
-  - name: metal3-centos-e2e-feature-test-main
+  # name: {job_prefix}-{image_os}-e2e-feature-test-{capm3_target_job}
```

Comment:
```suggestion
  # name: {job_prefix}-{image_os}-e2e-feature-test-{capm3_target_branch}-{capm3_target_job}
```

Reply:
It is not necessary, as `capm3_target_job` values also pointed as `main-pivoting` , `main-remediation` ...... So branch name is covered here.